crates/wit-bindgen/src/lib.rs

```diff
@@ -2960,6 +3046,160 @@ impl<'a> RustGenerator<'a> for InterfaceGenerator<'a> {
     }
 }
 
+#[derive(Default)]
+struct LinkOptionsBuilder {
+    unstable_features: HashSet<String>,
```

Comment:
Could this use a `BTreeSet` or `IndexSet` to ensure it always generates a deterministic order of options?

Reply:
Definitely